### ISSUE TITLE
Output quantity JSON to hot phase module

### ISSUE THREAD

**Issue by angaither** [DoSomethingArchive/legacy-website#4607]
return a json object to the campagin in hot module phases with the quantity total for each day

```
select quanity 
from dosomething_quanity_log
where nid = X
group timestamp by dates
and n.timestamp is max where timestamps 
```

**Comment by mikefantini** [comment]
@angaither this is so we can (more) easily populate the graph yeah?

**Comment by angaither** [comment]
@mikefantini yup

**Comment by sbsmith86** [comment]
@angaither All I need for the graph is 4 data points. One for each day we are showing on the x-axis. Is it possible to have a JSON object that holds just the four dates we want to show on the graph, and the goal progress on each of those dates (set as null if the date is in the future). So something like: 

```
{
  dates: ["JAN 1", "JAN 10", "JAN 21", "JAN 31"],
  value: [3000, 4500, 9000, null],
}
```

**Comment by angaither** [comment]
@sbsmith86 hmm, originally when we talked about this we were going to send the latest timestamp from each date, because how do we pick which four dates we are going to use?
What happens when we run a campaign for a week?

**Comment by sbsmith86** [comment]
@angaither yea, we still need that data to aggregate it into these 4 data points because the graph only shows 4 points. Could we determine the dates by getting using the hot phase start and end dates, divide by 4, and then calculate dates based on that interval? If the hot phase is only for a week, then we could still determine 4 dates in between start and end.

**Comment by angaither** [comment]
oh boy

**Comment by mikefantini** [comment]
@sbsmith86 can we sync on this? I thought we were going to be adding to the graph regularly, but only having markers at those 4 points in the date range (i.e. 4 markers within the one month).

I think the clean lines were just for ease on @lkpttn's part

![image](https://cloud.githubusercontent.com/assets/6330971/8432331/673ff5dc-1f0d-11e5-80eb-e4d8924cd1ec.png)

**Comment by sbsmith86** [comment]
@mikefantini Ah yea, lets sync. You are right and I think I confused things talking about aggregating stuff. But regardless, I definitely need data about those four dates, so there will be some logic around figuring out those dates programmatically, either on the backend or the frontend.

**Comment by mshmsh5000** [comment]
I think we can keep this on the front end. If you've got the and date and start date, you can divide by four and round up to get the two interim dates.

**Comment by sbsmith86** [comment]
Just wrapping up this convo. I made a custom graph type that will allow us to go ahead and pull in all of the data for each day of the month like originally planned!

**Comment by angaither** [comment]
:+1: awesome @sbsmith86 I'm working on outputting this data now!

**Comment by angaither** [comment]
Fixed by #4664
